ext/pdo_mysql/tests/pdo_mysql_stmt_nextrowset.phpt

```diff
@@ -33,10 +33,6 @@ if (!MySQLPDOTest::isPDOMySQLnd())
     if (false !== ($tmp = $stmt->nextRowSet()))
         printf("[002] Expecting false got %s\n", var_export($tmp, true));
 
-    // TODO: should give a warning, but its PDO, let's ignore the missing warning for now
-    if (false !== ($tmp = $stmt->nextRowSet(1)))
-        printf("[003] Expecting false got %s\n", var_export($tmp, true));
```

Comment:
Hm, why did this have to change?

Reply:
Because it's a ZPP test that only 0 arguments are passed, so it's kinda unrelated to PDO

Reply:
Ah right. In that case I don't get how this test was passing before?

Reply:
It's XFAIL I think